tasks/collect-collectors-data/tests/test-collect-collectors-data-fail-missing-cr.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-collect-collectors-data-fail-missing-cr
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: |
+    Run the collect-collectors-data task without a ReleasePlanAdmission cr and verify that the task fails as expected.
+    The `get-resource` command doesn't fail when a jsonpath is provided, it just returns {} if
```

Comment:
"verify the task fails"
"The command doesn't fail"

?? what is failing the task then?

Reply:
I don't think this makes any sense here. Removed.